gui/re.chiaki.Chiaki.appdata.xml

```diff
@@ -14,9 +14,12 @@
       Chiaki is a Free and Open Source Client for PlayStation 4 Remote Play. It can be used to play in real time on a PlayStation 4 as long as there is a network connection.
     </p>
     <p>
-      For more information and instructions how to use Chiaki, please refer to the official documentation: https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage
+      For more information and instructions how to use Chiaki, please refer to the official documentation: <url type="help">https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage</url>
     </p>
   </description>
+  <releases>
+    <release version="1.3.0" date="2020-10-18"/>
```

Comment:
Please, **do not include releases** in the upstream appdata file! I intentionally did not include them because otherwise @thestr4ng3r will have to manually update this file **every time** before a new Chiaki release. Outdated release info is useless.

Reply:
Release is required tag... being up-to-date is not required...  I'm pretty sure @thestr4ng3r will automate that easily in his release script.

Reply:
It is not required by most AppStream parsers/validators. Feel free to try validating it using appstream-util/appstreamcli. Anyway, the AppStream file was [removed](https://github.com/thestr4ng3r/chiaki/pull/358#issuecomment-719533680). Feel free to use the old AppStream file (that I made) in Debian and maintain it there.